### ISSUE TITLE
Apache Not Starting On Restart

### ISSUE THREAD

**Issue by mikewhitby** [mikewhitby/magento-ansible#4]
Seems that apache is not starting after performing a `vagrant reload`, or `vagrant halt` then `vagrant up`.

**Comment by mikewhitby** [comment]
The issue is that `vagrant-bindfs` binds after booting (I didn't realise this), so options seems to be:
- **Stop using vagrant-bindfs** - i.e install bindfs, and add an fstab mapping, via ansible. I'm not sure this would work though, because NFS exports are made after boot, so the bindfs mapping would be made before the NFS share existed
- **Get apache to start with nonexistent document root** - not sure if this can be done, a quick Google says no
- **Symlink the document root** - this might fool apache in to starting even though the symlink target doesn't exist, I've not tried it
- **Start apache after vagrant-bindfs is done** - not sure how, vagrant could maybe do this, or some sort of horrible cron job every minute? Yuk

Will take a look.

**Comment by mikewhitby** [comment]
Symlinks don't work, we need bindfs because the NFS shares get mounted post-boot, so that's options 1 and 3 out the window. The only reason apache won't start is because it does a configtest before starting, so I think the best bet is to skip the configtest and just start. Otherwise, could make a shell provisioner which starts apache, but that sucks a bit

**Comment by mikewhitby** [comment]
It's just the access and error logs that stop it from starting, I've tried symlinking and hardlinking them but neither work, probably because that just doesn't make any sense (because we're exporting to the guest, not from it), and the logs were a 'nice-to-have', so I'll just ditch them. I'll ditch all logs actually, no point having just the PHP log. Maybe I'll look at getting them back sometime, but I doubt it.
